Sources/Orbit/Support/Components/TimelineItem.swift

```diff
@@ -0,0 +1,413 @@
+import SwiftUI
+
+public extension VerticalAlignment {
```

Comment:
Can be internal and moved below after view definition, near typ extensions (example: `List` component, there we actually want the alignment to be available to align things below horizontally)